cli/src/commands/__tests__/install-test.js

```diff
@@ -1488,6 +1488,295 @@ declare type jsx$HTMLElementProps = {||}`;
         });
       });
     });
+
+    describe('definitions with dependencies', () => {
+      it('installs a dependency of appropriate version if not installed by project', () => {
+        return fakeProjectEnv(async FLOWPROJ_DIR => {
+          // Create some dependencies
+          await Promise.all([
+            mkdirp(path.join(FLOWPROJ_DIR, 'src')),
+            writePkgJson(path.join(FLOWPROJ_DIR, 'package.json'), {
+              name: 'test',
+              devDependencies: {
+                'flow-bin': '^0.43.0',
+              },
+              dependencies: {
+                'has-dep': '1.2.3',
+              },
+            }),
+            mkdirp(path.join(FLOWPROJ_DIR, 'node_modules', 'flow-bin')),
+            mkdirp(path.join(FLOWPROJ_DIR, 'node_modules', 'has-dep')),
+          ]);
+
+          await touchFile(path.join(FLOWPROJ_DIR, 'src', '.flowconfig'));
+          await mkdirp(path.join(FLOWPROJ_DIR, 'src', 'flow-typed'));
+
+          // Run the install command
+          await run({
+            ...defaultRunProps,
+            rootDir: path.join(FLOWPROJ_DIR, 'src'),
+          });
+
+          // Installs libdef
+          expect(
+            await fs.exists(
+              path.join(
+                FLOWPROJ_DIR,
+                'src',
+                'flow-typed',
+                'npm',
+                'foo_v1.x.x.js',
```

Comment:
Yep that's right, so we're checking that even though it's not declared in the projects package.json it will still appear in the flow-typed dir after install. It's references here in the fixtures https://github.com/flow-typed/flow-typed/pull/4331/files#diff-1037410880ad59c3a8d21833993d5e979a72f0d42a96c97ea3a84dc0d0c67e7d